quattor/aii/opennebula/rpms.pan

```diff
@@ -0,0 +1,22 @@
+# #
+# Software subject to following license(s):
+#   Apache 2 License (http://www.opensource.org/licenses/apache2.0)
+#   Copyright (c) Responsible Organization
+#
+
+# #
+# Current developer(s):
+#   Alvaro Simon Garcia <Alvaro.SimonGarcia@UGent.be>
+#   Stijn De Weirdt <Stijn.DeWeirdt@UGent.be>
+#
+
+# 
+# #
+# aii-opennebula, 16.2.0-rc4-SNAPSHOT, rc4_SNAPSHOT20160219150415, 2016-02-19T15:04:15Z
+#
+
+# Template adding aii-opennebula rpm to the configuration
+
+unique template quattor/aii/opennebula/rpms;
+
+"/software/packages"=pkg_repl("aii-opennebula","16.2.0-rc4_SNAPSHOT20160219150415","noarch");
```

Comment:
ok, these templates should come together with the other aii templates, for a specific release I guess. @jrha  don't know why they are not here yet, aii-opennebula is in master?